src/journal_send.cc

```diff
@@ -1,14 +1,28 @@
 // Down below you will find the most intuitive code you probably ever saw on the
 // Internet. It just copies an ordinary JavaScript string array into an iovec
 // and then finally calls sd_journal_sendv. Rocket science included!
+// Priority is a special case as it needs to be taken from the syslog.h
+// preprocessor definitions, which aren't available from JavaScript.
 
 #include <nan.h>
 #include <systemd/sd-journal.h>
+#include <syslog.h>
+
+#define to_str_h(x) "" #x
+#define to_str(x) to_str_h(x)
+
+void setIovec( struct iovec *iov, const char *priorityStr ) {
+	size_t strLength = strlen(priorityStr) + 9;
+	iov->iov_len = strLength;
+	iov->iov_base = (char*) malloc( strLength + 1 );
+	snprintf((char*) iov->iov_base, strLength + 1, "PRIORITY=%s", priorityStr);
```

Comment:
Technically, the integer could be 12345 or something - so we don't know the length.  I thought it was cleaner to use strings, because then we can use strlen() and malloc() exactly the right amount of memory to fit in the integer part of the string.